src/components/pill-selector-dropdown/PillSelector.scss

```diff
@@ -50,6 +50,10 @@
             border-color: $primary-color;
             outline: 0;
             box-shadow: none;
+
+            .bdl-PillSelector-input.input-in-new-line {
```

Comment:
we should decrease the specificity if possible. otherwise it could be hard to override in the future